CONTRIBUTING.md

```diff
@@ -44,6 +44,13 @@ dependencies.
    Hence, to run these tests, you will need Python 2.7 and Python 3.x
    (preferably 3.6+) interpreters installed on your system.
 
+Required Python packages are listed in [requirements.txt](requirements.txt) file
+in this repository. This can be done with pip with the following command:
+
+```
+$> pip install -r requirements.txt
```

Comment:
nit: Please format this as:

\`\`\`shell
pip install -r requirements.txt
\`\`\`

to match the rest of our docs.

Reply:
Done. Would you like me to also update the rest of the shell parts in this file while I'm at it? I was trying to follow the style in this particular file.

Reply:
Ah, thanks for pointing that out; I hadn't noticed. I'll update the rest of the formatting in a follow-up change.